scipy/optimize/_linprog_util.py

```diff
@@ -1086,6 +1086,79 @@ def vstack(blocks):
     return A, b, c, c0, x0
 
 
+def _round_to_power_of_two(x):
+    """
+    Round elements of the array to the nearest power of two.
+    """
+    return 2**np.round(np.log2(x))
```

Comment:
:+1: This rounds towards *-inf* to the nearest power of two matching the behaviour of Python/NumPy's ``round`` functions.

Reply:
Hmm didn't remember that. I actually changed to `around`. Not sure that it really matters, but I suppose `around` is what I had in mind. Thanks.